src/components/Tabnav.vue

```diff
@@ -67,7 +67,7 @@ $-tabnav-top-margin: rem(15px);
 $-tabnav-bottom-margin: rem(25px);
 
 .tabnav {
-  margin: $-tabnav-top-margin 0 $-tabnav-bottom-margin 0;
+  margin: 0 0 $-tabnav-bottom-margin 0;
```

Comment:
`TabNavigator` and `EndpointExample` both handle the top spacing. No need to have that here.